src/components/Picker/index.js

```diff
@@ -102,10 +104,28 @@ class Picker extends PureComponent {
         };
 
         this.onInputChange = this.onInputChange.bind(this);
+        this.placeholder = this.props.placeholder;
+        this.items = this.props.items;
     }
 
     componentDidMount() {
         this.setDefaultValue();
+
+        // Windows will reuse the text color of the select for each one of the options
+        // so we might need to color accordingly so it doesn't blend with the background.
+        if (getOperatingSystem() === CONST.OS.WINDOWS) {
+            this.placeholder = _.isEmpty(this.placeholder) ? {} : {
+                ...this.placeholder,
+                color: '#002140',
```

Comment:
For future reference, in case someone stumbles on this PR, we didn't create a constant for this because we'll have to clean it up anyways when the light theme is up soon.